Sj zig vopr (#98)
tigerbeetle/tigerbeetle: README.md

To run TigerBeetle's long-running simulation, called *The VOPR*:

```bash
zig/zig build vopr
```

> Add the send flag to enable the VOPR to report bugs to the VOPR Hub which can automatically create GitHub issues as needed.

Reviewer: ```suggestion
Pass the `send` flag if you want the VOPR also to report any bugs to the VOPR Hub, which can replay, deduplicate, and automatically create GitHub issues as needed.
```

> Pass the `send` flag if you want the VOPR also to report any bugs to the VOPR Hub, which can replay, deduplicate, and automatically create GitHub issues as needed.

Reviewer 2: ```suggestion
Pass the `--send` flag to the VOPR to report discovered bugs to the VOPR Hub. The VOPR Hub will automatically replay, deduplicate, and create GitHub issues as needed.
```

> Pass the `send` flag if you want the VOPR also to report any bugs to the VOPR Hub, which can replay, deduplicate, and automatically create GitHub issues as needed.

Reviewer 2: Also, since this is the first mention of the VOPR Hub, turn "VOPR Hub" (in the text) into a link to the VOPR Hub's README.

```bash

> zig/zig build vopr -- --send

Reviewer: It would be great to also have an example here in the README of running a specific seed or getting help to see all the available command line arguments.
